src/config.ts

```diff
@@ -615,3 +599,26 @@ export const IO_WALLET_API_CLIENT = IoWalletAPIClient(
 export const FF_IO_WALLET_ENABLED = process.env.FF_IO_WALLET_ENABLED === "1";
 export const FF_IO_WALLET_TRIAL_ENABLED =
   process.env.FF_IO_WALLET_TRIAL_ENABLED === "1";
+
+export const FF_IO_X_USER_TOKEN = pipe(
+  process.env.FF_IO_X_USER_TOKEN,
+  FeatureFlag.decode,
+  E.getOrElse(() => FeatureFlagEnum.NONE)
+);
+export const FF_IO_X_USER_TOKEN_BETA_TESTER_SHA_LIST = pipe(
+  process.env.FF_IO_X_USER_TOKEN_BETA_TESTER_SHA_LIST,
+  CommaSeparatedListOf(NonEmptyString).decode,
+  E.getOrElseW((errs) => {
+    log.error(
+      `Missing or invalid FF_IO_X_USER_TOKEN_BETA_TESTER_SHA_LIST environment variable: ${readableReport(
+        errs
+      )}`
+    );
+    return process.exit(1);
+  })
+);
```

Comment:
This change will make the ENV required to start successfully the application. We need to set this value before merge and deploy this PR. Make the PR Draft with `no-merge` tag